cmd/hyperconverged-cluster-operator/main.go

```diff
@@ -32,6 +31,7 @@ import (
 	apiruntime "k8s.io/apimachinery/pkg/runtime"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	cdiv1beta1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1beta1"
+	sspv1alpha1 "kubevirt.io/ssp-operator/api/v1alpha1"
```

Comment:
I fear we can accept it only if >= v1beta1

Reply:
What is the criteria to move from v1alpha1 to v1beta1? Or this is merely a syntactical requirement?

Reply:
The criteria is:

Since it is a "beta" suffixed, it should work with subsequent releases. Indeed, the meaning of api versions is as following:

alpha
API versions with ‘alpha’ in their name are early candidates for new functionality coming into Kubernetes. These may contain bugs and are not guaranteed to work in the future.

beta
‘beta’ in the API version name means that testing has progressed past alpha level, and that the feature will eventually be included in Kubernetes. Although the way it works might change, and the way objects are defined may change completely, the feature itself is highly likely to make it into Kubernetes in some form.

stable
These do not contain ‘alpha’ or ‘beta’ in their name. They are safe to use.

an operator is required to have it's CRD at least to v1beta1 to be considered GA as our previous version was.

Reply:
The API version of `ssp-operator` is now `v1beta1`.

Reply:
@akrejcir thanks, will update this PR to consume `v1beta1`.